client/client.go

```diff
@@ -513,9 +513,9 @@ type (
 		// GetRunID() will always return "run ID 1" and  Get(ctx context.Context, valuePtr interface{}) will return the result of second run.
 		GetWorkflow(ctx context.Context, workflowID string, runID string) WorkflowRun
 
-		// SignalWorkflow sends a signals to a workflow in execution
+		// SignalWorkflow sends a signals to a running workflow.
 		// - workflow ID of the workflow.
-		// - runID can be default(empty string). if empty string then it will pick the running execution of that workflow ID.
+		// - runID can be default(empty string). If empty string then it will pick the running execution of that workflow ID.
```

Comment:
```suggestion
		// - runID can be default(empty string). If set to the empty string, then it will pick the running execution of that workflow ID.
```

Reply:
Taken. (Although I dropped the article -- I reckon "the empty string" only makes sense to mathematicians!)